src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -1,6 +1,174 @@
+For ASP.NET applications, this should be in `Global.aspx.cs`.
+
+### GenevaExporterOptions (for logs and traces)
+
+`GenevaExporterOptions` contains various options to configure the Geneva
+Exporter.
+
+#### `ConnectionString` (required)
+
+On Linux the connection string has the format `Endpoint=unix:{UDS Path}`.
+
+On Windows the connection string has the format `EtwSession={ETW session}`.
+
+#### `CustomFields` (optional)
+
+A list of fields which should be stored as individual table columns.
+
+#### `PrepopulatedFields` (optional)
+
+This is a collection of fields that will be applied to all the logs and traces
+sent through this exporter.
+
+#### `TableNameMappings` (optional)
+
+This defines the mapping for the table name used to store traces and logs.
+
+The default table name used for traces is `Span`. For changing the table name
+for traces, add an entry with key as `Span`, and value as the custom table name.
+
+The default table name used for logs is `Log`. Mappings can be specified for
+each
+[category](https://docs.microsoft.com/dotnet/core/extensions/logging#log-category)
+of the log. For changing the default table name for logs, add an entry with key
+as `*`, and value as the custom table name.
+
+### Enable Metrics
+
+**NOTE:** _Version 1.2.x of the Geneva Exporter (which adds Metrics support)
+does not support .NET versions lower than .NET Framework 4.6.1._
+
+This snippet shows how to configure the Geneva Exporter for metrics
+
+```csharp
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter("TestMeter")
+    .AddGenevaMetricExporter(options =>
+    {
+        options.ConnectionString = "Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace";
+    })
+    .Build();
+```
+
+The above code must be in application startup. In case of ASP.NET Core
+applications, this should be in `ConfigureServices` of `Startup` class.
+For ASP.NET applications, this should be in `Global.aspx.cs`.
+
+### GenevaMetricExporterOptions (for metrics)
+
+`GenevaMetricExporterOptions` contains various options which are required to
+configure the GenevaMetricExporter.
+
+#### `ConnectionString` (required for metrics)
+
+On Windows **DO NOT** provide an ETW session name for Metrics, only specify
+Account and Namespace. For example:
+`Account={MetricAccount};Namespace={MetricNamespace}`.
+
+On Linux provide an `Endpoint` in addition to the `Account` and `Namespace`.
+For example:
+`Endpoint=unix:{UDS Path};Account={MetricAccount};Namespace={MetricNamespace}`.
+
+#### `MetricExportIntervalMilliseconds` (optional)
+
+Set the exporter's periodic time interval to export metrics. The default value
+is 20000 milliseconds.
+
+#### `PrepopulatedMetricDimensions` (optional)
+
+This is a collection of the dimensions that will be applied to _every_ metric
+exported by the exporter.
+
+## Troubleshooting
+
+Before digging into a problem, check if you hit a known issue by looking at the
+[CHANGELOG.md](./CHANGELOG.md) and [GitHub
```

Comment:
Fix the link to the CHANGELOG.

Reply:
When is the link not working? Works from GitHub's rendering of this file